packages/data/src/macros/macros_mm.yml

```diff
@@ -31,6 +31,10 @@
 "has_tunic_goron": "has_tunic_goron_strict || trick(MM_TUNICS)"
 "has_tunic_zora": "has_tunic_zora_strict || trick(MM_TUNICS)"
 
+# Scales
+"can_dive_small": "has_scale_raw(1)"
+"can_dive_big": "has_scale_raw(2)"
```

Comment:
Could maybe include ` || underwater_walking` with these, like how OOT's include Iron Boots. Though would then need to remove underwater_walking in every or clause that has them both.

Reply:
I was considering that, but decided not to do that. I like it how it is.